src/components/SubmissionsPage/Submissions.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import { Link } from 'gatsby';
+
+const useStyles = makeStyles(theme => ({
+	title: {
+		fontWeight: theme.typography.fontWeightBold,
+		paddingTop: theme.spacing(11),
+		[theme.breakpoints.down('sm')]: {
+			paddingTop: theme.spacing(4)
+		},
+		paddingBottom: theme.spacing(4)
+	},
+	info: {
+		paddingTop: theme.spacing(4)
+
+	},
+	divider: {
+		height: '2px',
+		marginLeft: theme.spacing(4),
+		marginRight: theme.spacing(4),
+		marginBottom: theme.spacing(4)
+	}
+}));
+
+
+const SubmissionsPage = () => {
+	const classes = useStyles();
+	return <Container maxWidth='md'>
+		<Typography align='left' variant='h4' component='h1' className={classes.title}>
+			Submission Info
+		</Typography>
+
+		<Typography align='left' variant='h5' component='h2'>
+			Thank You for Participating in HOTH XI!
+		</Typography>
+
+		<Typography align='left' variant='h5' component='h2'>
```

Comment:
It's a little odd to have 2 lines so close to each other that have the same styling, like this. Maybe we can move the `Thank you` text to the bottom after the steps to submit and make it `variant='h6' component='h3'` or something?